addons/sale_timesheet/report/timesheets_analysis_report.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from odoo.addons.sale_timesheet.models.account import TIMESHEET_INVOICE_TYPES
+
+class TimesheetsAnalysisReport(models.Model):
+
+    _inherit = "timesheets.analysis.report"
+
+    so_line = fields.Many2one("sale.order.line", string="SO line", readonly=True)
+    timesheet_invoice_type = fields.Selection(TIMESHEET_INVOICE_TYPES, string="Billable Type", readonly=True)
+    timesheet_revenues = fields.Float("Timesheet Revenues", readonly=True, help="Number of hours spent multiplied by the unit price per hour/day.")
+    margin = fields.Float("Margin", readonly=True, help="Timesheets revenues minus the costs")
+
+    billable_time = fields.Float("Billable Hours", readonly=True, help="Number of hours/days linked to a SOL.")
```

Comment:
You can remove this blank line. ^^
```suggestion
    billable_time = fields.Float("Billable Hours", readonly=True, help="Number of hours/days linked to a SOL.")
```